opentelemetry-sdk/tests/metrics/integration_test/test_exponential_bucket_histogram.py

```diff
@@ -0,0 +1,305 @@
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from platform import system
+from unittest import TestCase
+
+from pytest import mark
+
+from opentelemetry.sdk.metrics import Histogram, MeterProvider
+from opentelemetry.sdk.metrics.export import (
+    AggregationTemporality,
+    InMemoryMetricReader,
+)
+from opentelemetry.sdk.metrics.view import (
+    ExponentialBucketHistogramAggregation,
+)
+
+
+class TestExponentialBucketHistogramAggregation(TestCase):
+
+    test_values = [2, 4, 1, 1, 8, 0.5, 0.1, 0.045]
+
+    @mark.skipif(
+        system() == "Windows",
+        reason=(
+            "Tests fail because Windows time_ns resolution is too low so "
+            "two different time measurements may end up having the exact same"
+            "value."
+        ),
+    )
+    def test_synchronous_delta_temporality(self):
+
+        aggregation = ExponentialBucketHistogramAggregation()
+
+        reader = InMemoryMetricReader(
+            preferred_aggregation={Histogram: aggregation},
+            preferred_temporality={Histogram: AggregationTemporality.DELTA},
+        )
+
+        provider = MeterProvider(metric_readers=[reader])
+        meter = provider.get_meter("name", "version")
+
+        histogram = meter.create_histogram("histogram")
+
+        results = []
+
+        for _ in range(10):
+
+            results.append(reader.get_metrics_data())
+
+        for metrics_data in results:
+            self.assertIsNone(metrics_data)
+
+        results = []
+
+        for test_value in self.test_values:
+            histogram.record(test_value)
+            results.append(reader.get_metrics_data())
+
+        metric_data = (
```

Comment:
I think it is actually more clear to assert specifically the parts of interest, since that shows clearly what valuese are we interested in being equal.